### PR TITLE
front end  - bring back asset drop down in transfers 

### PR DESCRIPTION
**Pull request by easteregg**
## Issue
https://app.clickup.com/t/2yuzajm

## Description
*Please replace this line with info that is not in the ClickUp ticket, eg: "also bumps dependency foo version"*

## Checklist

- [ ] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_
- [ ] I have updated the `book/` to reflect changes made by this PR _(if applicable)_

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2yuzajm Front end - bring back asset drop down in transfers](https://app.clickup.com/t/2yuzajm)

File: frontend/apps/picasso/components/Molecules/BigNumberInput/hooks.ts [hunk 1]
```diff
@@ -82,6 +86,9 @@ export function useValidation({
       if (eventValue.match(NUMBERS_ONE_DOT) || !eventValue.length) {
         setStringValue(eventValue);
         setValid(false);
+      } else {
+        setStringValue(eventValue);
+        setValid(false);
       }
```
Comment (andresmechali) [inline frontend/apps/picasso/components/Molecules/BigNumberInput/hooks.ts:92]:
```suggestion
      setStringValue(eventValue);
      setValid(false);
```

File: frontend/apps/picasso/stores/defi/polkadot/balances/slice.ts [hunk 2]
```diff
@@ -85,7 +85,7 @@ export interface SubstrateBalancesSlice {
 
 export const createSubstrateBalancesSlice: StoreSlice<
   SubstrateBalancesSlice
-> = (set) => ({
+> = (set, get) => ({
```
Comment (andresmechali) [inline frontend/apps/picasso/stores/defi/polkadot/balances/slice.ts:88]: this `get` is not being used, is it?

File: frontend/apps/picasso/stores/defi/polkadot/transfers/utils.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+export function getDefaultToken(tokenOptions: Array<TokenOption>): AssetId {
+  const found = tokenOptions.find((token) => !token.disabled);
+  if (found) {
+    return found.tokenId;
+  } else if (tokenOptions.length === 1) {
+    return tokenOptions[0].tokenId;
+  }
+
+  return tokenOptions[0].tokenId;
```
Comment (andresmechali) [inline frontend/apps/picasso/stores/defi/polkadot/transfers/utils.ts:12]:
```suggestion
  } 

  return tokenOptions?.[0]?.tokenId;
```

**Review by andresmechali — APPROVED** [review]
LGTM 🚀

File: frontend/apps/picasso/defi/polkadot/pallets/xcmp.ts [hunk 4]
```diff
@@ -380,27 +369,27 @@ export async function transferPicassoKusama({
     signerAddress,
     api,
     signer,
-    txHash => {
+    (txHash) => {
       enqueueSnackbar("Transfer executed", {
```
Comment (saadjhk) [inline frontend/apps/picasso/defi/polkadot/pallets/xcmp.ts:373]: @easteregg we should add subscan links here as well

**Review by saadjhk — APPROVED** [review]

**Pull request merged**